src/components/StructuredLimitations.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import _ from 'lodash';
+import { div, span, a, i, hh } from "react-hyperscript-helpers";
+import { Theme } from '../libs/theme';
+import { Files } from '../libs/ajax';
+import { download } from '../libs/utils';
+
+const ROOT = {
+  fontFamily: 'Montserrat',
+  color: Theme.palette.primary,
+  whiteSpace: 'pre-line'
+};
+
+const HEADER = {
+  marginBottom: '5px',
+  fontSize: Theme.font.size.header,
+  lineHeight: Theme.font.leading.regular,
+  fontWeight: Theme.font.weight.semibold,
+  color: Theme.palette.hightlighted,
+};
+
+const TEXT = {
+  fontSize: Theme.font.size.small,
+  lineHeight: Theme.font.leading.regular,
+  fontWeight: Theme.font.weight.regular,
+};
+
+const BOLD = {
+  ...TEXT,
+  fontWeight: Theme.font.weight.semibold,
+};
+
+const ICON = {
+  color: Theme.palette.link,
+  marginRight: '6px',
+};
+
+export const StructuredLimitations = hh(class StructuredLimitations extends React.PureComponent {
+  /**
+   * converts the given string to the desired use restrictions object
+   * this will be deleted once we implement returning the JSON object from the API
+   */
+  toObject = content => {
+    const lines = content.split('\n');
+    lines.shift(); // gets rid of that "Samples are restricted..." line
+    const formatObject = {
+      primary: [],
+      secondary: []
+    };
+
+    lines.forEach(line => {
+      let code, description;
+      const { primary, secondary } = formatObject;
+      const primaryCodes = ["NRES", "GRU", "HMB", "DS", "POA"];
```

Comment:
This looks correct to me. I'll dig in a little more to be sure, but for now, this is perfect.